docs/src/pages/components/alert/react-auto-prop-table-alert.mdx

```diff
@@ -0,0 +1,119 @@
+import { Expander, ExpanderItem } from '@aws-amplify/ui-react';
```

Comment:
A general thought to the team: should we check in these generated files? Checking them in increases our repo size, but does allow for inspection during a PR. We could add them to gitignore and have like a `props/` director in each component and ignore that directory (when we build angular and vue primitives as well). I don't have a super strong opinion one way or the other, just wanted to bring it up for discussion.

Reply:
We build them eventually...so does the size really matter when checking in?

Reply:
Done